scripts/generate-examples.py

```diff
@@ -111,7 +111,7 @@
                 options.update(metaOptions)
 
             # If the example has additional options, load them.
-            example_options: str = example.get("options", {})
+            example_options: Dict = example.get("options")
```

Comment:
Hmm... this isn't really correct.

`.get()` returns `None` if the second option isn't specified. If you want it to always return a dictionary, you need the `example.get("options", {})`. Otherwise, you should annotate the variable to say that it can be a none type. So, either:

```
example_options: Dict = example.get("options", {})
```

or

```
example_options: Optional[Dict] = example.get("options")
```

Annotating the variable would mean that your IDE could catch things like:

```
example_options["transpose"]
```

Since this *could* be None, and you can't do an index lookup on None.